.github/workflows/ferretdb_packages.yml

```diff
@@ -52,20 +52,27 @@ concurrency:
   cancel-in-progress: false
 
 jobs:
-  deb:
+  packages:
     strategy:
       fail-fast: false
       matrix:
-        os: [deb11, deb12, ubuntu22.04, ubuntu24.04]
+        os: [deb11, deb12, ubuntu22.04, ubuntu24.04, rhel8, rhel9]
         arch: [amd64, arm64]
         pg: [15, 16, 17]
         include:
           - arch: amd64
             runner: ubuntu-24.04
           - arch: arm64
             runner: ubuntu-24.04-arm
-
-    name: .debs (${{ matrix.os }}, ${{ matrix.arch }}, Pg${{ matrix.pg }})
+        exclude:
+          # pg 15 is not supported for rpm packages
+          # https://github.com/FerretDB/documentdb/blob/ferretdb/packaging/README.md
+          - os: rhel8
+            pg: 15
+          - os: rhel9
+            pg: 15
```

Comment:
Also it seems it's `arm` is _propably_ not supported https://github.com/FerretDB/documentdb/blob/ferretdb/.github/workflows/build_rpm_packages.yml#L33

I think requested arch is `amd` only https://github.com/microsoft/documentdb/issues/177.